aspnetcore/toc.md

```diff
@@ -229,85 +207,111 @@
+### [Continuous deployment with Azure Pipelines](/azure/devops/pipelines/get-started-yaml)
+### [Troubleshoot on Azure App Service](xref:host-and-deploy/azure-apps/troubleshoot)
+## Azure DevOps 
 ### [Introduction](xref:azure/devops/index)
 ### [Tools and downloads](xref:azure/devops/tools-and-downloads)
 ### [Deploy to App Service](xref:azure/devops/deploy-to-app-service)
 ### [Continuous integration and deployment](xref:azure/devops/cicd)
 ### [Monitor and troubleshoot](xref:azure/devops/monitor)
 ### [Next steps](xref:azure/devops/next-steps)
-
-# [Client-side development](xref:client-side/index)
-## [Use Gulp](xref:client-side/using-gulp)
-## [Use Grunt](xref:client-side/using-grunt)
-## [Use LibMan](xref:client-side/libman/index)
-### [LibMan CLI](xref:client-side/libman/libman-cli)
-### [LibMan in Visual Studio](xref:client-side/libman/libman-vs)
-## [Manage client-side packages with Bower](xref:client-side/bower)
-## [Style apps with LESS, Sass, and Font Awesome](xref:client-side/less-sass-fa)
-## [Bundle and minify](xref:client-side/bundling-and-minification)
-## [Use Browser Link](xref:client-side/using-browserlink)
-## [Use JavaScriptServices for SPAs](xref:client-side/spa-services)
-## [Use the SPA project templates](xref:spa/index)
-### [Angular project template](xref:spa/angular)
-### [React project template](xref:spa/react)
-### [React with Redux project template](xref:spa/react-with-redux)
-
-# [Mobile](xref:mobile/index)
-## [Create backend services for native mobile apps](xref:mobile/native-mobile-backend)
-
-# [Host and deploy](xref:host-and-deploy/index)
-## [Host on Azure App Service](xref:host-and-deploy/azure-apps/index)
-### [Publish to Azure with Visual Studio](xref:tutorials/publish-to-azure-webapp-using-vs)
-### [Publish to Azure with CLI tools](/azure/app-service/app-service-web-get-started-dotnet)
-### [Continuous deployment to Azure with Visual Studio and Git](xref:host-and-deploy/azure-apps/azure-continuous-deployment)
-### [Continuous deployment to Azure with Azure Pipelines](/azure/devops/pipelines/get-started-yaml)
-### [Troubleshoot ASP.NET Core on Azure App Service](xref:host-and-deploy/azure-apps/troubleshoot)
-## [Host on Windows with IIS](xref:host-and-deploy/iis/index)
-### [Troubleshoot ASP.NET Core on IIS](xref:host-and-deploy/iis/troubleshoot)
-### [ASP.NET Core Module configuration reference](xref:host-and-deploy/aspnet-core-module)
-### [Development-time IIS support in Visual Studio for ASP.NET Core](xref:host-and-deploy/iis/development-time-iis-support)
-### [IIS Modules with ASP.NET Core](xref:host-and-deploy/iis/modules)
+## Host on Windows with IIS
+### [Overview](xref:host-and-deploy/iis/index)
+### [Troubleshoot on IIS](xref:host-and-deploy/iis/troubleshoot)
+### [ASP.NET Core Module reference](xref:host-and-deploy/aspnet-core-module)
+### [Development-time IIS support in Visual Studio](xref:host-and-deploy/iis/development-time-iis-support)
```

Comment:
This could probably be shortened to "IIS support in Visual Studio". It's generally assumed you're doing development if using VS.